C++/Navio/Common/Ublox.cpp

```diff
@@ -346,6 +347,55 @@ int Ublox::testConnection()
     return 0;
 }
 
+int Ublox::configureSolutionRate(std::uint16_t meas_rate,
+                           std::uint16_t nav_rate,
+                           std::uint16_t timeref)
+{
+    CfgNavRate msg;
+    msg.measure_rate = meas_rate;
+    msg.nav_rate     = nav_rate;
+    msg.timeref      = timeref;
+
+    _sendMessage(CLASS_CFG, MSG_CFG_RATE, &msg, sizeof(CfgNavRate));
+}
+
+int Ublox::_sendMessage(std::uint8_t msg_class, std::uint8_t msg_id, void *msg, std::uint16_t size)
+{
+    unsigned char buffer[buffer_length];
```

Comment:
it's a const. Make it `UBX_BUFFER_LENGTH`

Reply:
Done